openapi-mock/package.json

```diff
@@ -37,7 +37,7 @@
     "url": "git@github.com:kyma-incubator/varkes.git"
   },
   "scripts": {
-    "compile": "tsc && cd src && copyfiles server/resources/* ../dist/ && copyfiles test/*.json ../dist/ && copyfiles test/*.yaml ../dist/ ",
+    "compile": "tsc && copyfiles -f -e '**/*.ts' src/test/* dist/test && copyfiles -f src/server/resources/* dist/server/resources",
```

Comment:
why are all the files in dist/test alone?

Reply:
I don't get the question. I simplified the command by simply copying all files of test folder excluding the ts files. With that new files placed there will be copied automatically.

Reply:
ah Ok I misunderstood the command